ReactWindows/ReactNative.Shared/UIManager/DependencyObjectExtensions.cs

```diff
@@ -14,10 +14,29 @@ namespace ReactNative.UIManager
     /// </summary>
     public static class DependencyObjectExtensions
     {
-        private static readonly ConcurrentDictionary<DependencyObject, DependencyObjectData> s_properties =
-            new ConcurrentDictionary<DependencyObject, DependencyObjectData>();
+        /// <summary>
+        /// Attached property for DependencyObjectData
+        /// </summary>
+        public static readonly DependencyProperty DependencyObjectExtensionsProperty =
+            DependencyProperty.RegisterAttached(
+                "DependencyObjectExtensions",
+                typeof(DependencyObjectData),
+                typeof(DependencyObject),
+                null);
+
         private static readonly IReactCompoundView s_defaultCompoundView = new ReactDefaultCompoundView();
 
+        private static DependencyObjectData GetData(this DependencyObject view, bool create = false)
```

Comment:
>create [](start = 85, length = 6)

I think it's worthwhile to have a `GetOrAdd` helper like we used to. It makes it clear what the method call will do.

```
Get(true)

// vs

GetOrAdd()
```

Without looking at Get's implementation, it's not obvious what `Get(true)` does.